DTML.EduBot/Dialogs/MeaningDialog.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace DTML.EduBot.Dialogs
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Bot.Builder.Dialogs;
+    using Microsoft.Bot.Builder.Luis;
+    using Microsoft.Bot.Builder.Luis.Models;
+    using DTML.EduBot.Common;
+    using DTML.EduBot.Qna;
+    using Microsoft.Azure;
+    using Attributes;
+    using System.Linq;
+    using DTML.EduBot.Constants;
+
+    [PreConfiguredLuisModel]
+    [PreConfiguredQnaModel]
+    [Serializable]
+    public partial class MeaningDialog : QnaLuisDialog<object>
```

Comment:
1. The Meaning Intent needs to be scoped down to handle utterances related to "what does XX mean". 
Meaning Intent Handler - should invoke bing search API for the queried entity and include results in 
a rich card format.
2. Remove - Sentence constructs, grammar 
3. Move out "how to pronounce" to a new intent. The respective intent handler should invoke the text to speech API